playground-3.js

```diff
@@ -1,3 +1,15 @@
 // https://miorita.scoalaweb.ro/playgrounds/escape
 // Scrie un program care să o ajute pe Miorița să ajungă la pășune
 // Programul trebuie să funcționeze indiferent de configurația stânei.
+
+let free = false;
+while (!free) {
+  if (canMove()) {
+    free = true;
+    while (canMove()) {
+      move();
+    }
+  } else {
+    turnRight();
+  }
+}
```

Comment:
Funcționează, dar mi se pare cam încâlcit.

Dacă te gândești puțin, cred că ai putea să faci același lucru mai simplu. Pont: Nu ai nevoie de variabilă.

Reply:
Bravo